StRoot/StEvent/StFcsCluster.h

```diff
@@ -61,15 +62,26 @@ class StFcsCluster : public StObject {
     void setChi2Ndf1Photon(float chi2ndfph1);
     void setChi2Ndf2Photon(float chi2ndfph2);
     void setFourMomentum(StLorentzVectorD p4);
+    //Hits
     StPtrVecFcsHit& hits();
     const StPtrVecFcsHit& hits() const;
+    //Neighbors
     void addNeighbor(StFcsCluster* neighbor);
     StPtrVecFcsCluster& neighbor();
     const StPtrVecFcsCluster& neighbor() const;    
+    //Points
     StPtrVecFcsPoint& points();
     const StPtrVecFcsPoint& points() const;
     void addPoint(StFcsPoint* p);
     void addPoint(StFcsPoint* p1, StFcsPoint* p2);
+
+    //Tracks
+    StPtrVecFwdTrack& tracks();
+    const StPtrVecFwdTrack& tracks() const;
+    void addTrack(StFwdTrack* p);
+    void sortTrackByPT();
```

Comment:
Why do you need to have sortTrackByPT() defined on StFcsCluster?    If you need the tracks sorted in some order,
just sort them... you already have the reference to the track vector...
void inSomeAnalysis() { 
auto& tracks = mCluster->tracks();
std::sort(tracks.begin(), tracks.end(), [](StFwdTrack* a, StFwdTrack* b) {
        return b->momentum().perp() < a->momentum().perp();
    });
}